hat/assets/js/apps/Iaso/domains/userRoles/components/CreateEditUserRole.tsx

```diff
@@ -38,6 +41,7 @@ export const CreateEditUserRole: FunctionComponent<Props> = ({
     isOpen,
     id,
     name,
+    editable_org_unit_type_ids,
```

Comment:
This is always empty for me… Is this a bug? (I hope I didn't break anything during my rebase)

![empty](https://github.com/user-attachments/assets/abd5d182-c813-43b5-a1eb-be6b6f980bc2)

Reply:
user_roles_editable_org_unit_type_ids=[[]], this should be fixed now